python-main.js

```diff
@@ -1460,6 +1461,12 @@ function web_editor(config) {
         };
 
         document.dispatchEvent(new CustomEvent('webusb', { detail: details }));
+        // If escape key is pressed close modal
+        $(document).keydown(function(e) {
+            if (e.which == 27) {
+                flashErrorClose();
+           }
+       });
```

Comment:
Why is this required here, if focusModal() at the beginning of this function does something similar?

Reply:
You can't use the escape key to close the error modal here like you can with vex modals, so it's included in order to do that. Should it be moved into `focusModal()`?

Reply:
In that case it should be part of modalMsg(), as that prepares the modal. focusModal() should just be about focusing the modal.